### DISCUSSION TITLE
Auto-Level makes no difference and Normalize not working with mogrify

### DISCUSSION THREAD

**Discussion by dumdoo** [ImageMagick/ImageMagick#2510] [Help]
``magick mogrify -format png -auto-level -channel RGB *.png``

I need to convert hundreds of files so i'm using _mogrify_. Originally as a test i was using `-normalize` with _convert_ but now it doesn't work and `-auto-level` hardly does anything. Any clue as to what's going wrong?

**Comment by fmw42** [comment]
With respect to -auto-level, your image probably already spans the full dynamic range. You can replace -auto-level with -contrast-stretch X% to stretch further. X=0 will produce something close to -auto-level.

With respect to -normalize, it works for me just fine with IM 7.0.10.28 Q16 Mac OSX Sierra (as does -contrast-stretch)
